### PR TITLE
Feature/email notifications

### LINKED ISSUE #124: As a user, i want to be notified via email when something happens related to my account
when creating a post or updating a circle, check if user has to be notified and send mail

### PR DESCRIPTION
**Pull request by KjellPirschel**
closes #124

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -304,3 +264,21 @@ def user_search(request):
 
     return JsonResponse({'suggestions': users_data}, safe=False)
 
+
+def email_notification_for_user(user, subject, templateFile, context={}):
```
Comment (Gjum) [inline users/views.py:268]: plz make documentation here!!!!

File: users/templates/users/notification_for_new_circle_email.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+<p>{{ content }}</p>
+
+<p>
+    Please make sure you take a look at circuit soon.
+    Wishing you only the best,<br/>
+    The circuit team
+</p>
+</body>
+</html>
```
Comment (metalmatze) [inline users/templates/users/notification_for_new_circle_email.html:13]: Add new line at EOF.

File: users/templates/users/notification_for_post_email.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+<a href={{ site_url }}{{ link_to_subject }}>To see the post click here.</a>
+
+
+<p>
+    Please make sure you take a look at circuit soon.
+    Wishing you only the best,<br/>
+    The circuit team
+</p>
+</body>
+</html>
```
Comment (metalmatze) [inline users/templates/users/notification_for_post_email.html:17]: Add new line at EOF.

File: users/views.py (hunk 1 above)
Comment (metalmatze) [inline users/views.py:268]: :+1:

**Pull request merged**